conf/defaults.ini

```diff
@@ -307,6 +307,9 @@ oauth_state_cookie_max_age = 600
 # limit of api_key seconds to live before expiration
 api_key_max_seconds_to_live = -1
 
+# Set to true to enable Sig4 authentication option for HTTP-based datasources
+sigv4_auth_enabled = true
```

Comment:
you have to add this configuration also in the [sample.ini](https://github.com/grafana/grafana/blob/master/conf/sample.ini#L277) so as users that copy this file for creating their custom configuration will be able to override it.